includes/wc-attribute-functions.php

```diff
@@ -136,8 +138,10 @@ function wc_attribute_label( $name, $product = '' ) {
 		if ( $product->is_type( 'variation' ) ) {
 			$product = wc_get_product( $product->get_parent_id() );
 		}
+		$attributes = $product->get_attributes();
```

Comment:
Should we maybe check first if `false !== $product` before running `$product->get_attributes()` to avoid a fatal error in case `wc_get_product()` returns false or null?